components/Tabs/src/ColorThemer/MDCTabBarColorThemer.m

```diff
@@ -29,6 +29,15 @@ + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
       [colorScheme.onPrimaryColor colorWithAlphaComponent:kUnselectedOpacity];
 }
 
++ (void)applySurfaceVariantWithColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```

Comment:
For chips the naming goes as `applySemanticColorScheme:toChoiceChip` for example. I am wondering if we should do the same here.